### PR TITLE
Add MkDocs specific Markdown guides to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Bas-Man**
This adds two links. These links are cheatsheets more specific to MkDocs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ Not sure what to write about, take a look at our [issues](https://github.com/sel
...
 ## Other Resources
 
-For help with Markdown:
-
-[Markdown -- Github Help](https://help.github.com/en/github/writing-on-github)
-
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+For help with Markdown:  
+[MkDocs Cheatsheet ](https://3os.org/markdownCheatSheet/welcome/)  
+[MkDocs -- Admonition](https://3os.org/markdownCheatSheet/admonition/)
```
Comment (RealOrangeOne) [inline CONTRIBUTING.md:103]: Probably worth linking to the `mkdocs-material` docs here instead https://squidfunk.github.io/mkdocs-material/reference/admonitions/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -98,8 +98,8 @@ Not sure what to write about, take a look at our [issues](https://github.com/sel
 
 ## Other Resources
 
-For help with Markdown:
-
-[Markdown -- Github Help](https://help.github.com/en/github/writing-on-github)
-
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+For help with Markdown:  
+[MkDocs Cheatsheet ](https://3os.org/markdownCheatSheet/welcome/)
```
Comment (RealOrangeOne) [inline CONTRIBUTING.md:102]:
```suggestion
[MkDocs Cheatsheet](https://3os.org/markdownCheatSheet/welcome/)  
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -98,8 +98,8 @@ Not sure what to write about, take a look at our [issues](https://github.com/sel
 
 ## Other Resources
 
-For help with Markdown:
-
-[Markdown -- Github Help](https://help.github.com/en/github/writing-on-github)
-
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+For help with Markdown:
```
Comment (RealOrangeOne) [inline CONTRIBUTING.md:101]: Maybe remove this heading?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bas-Man) [reply CONTRIBUTING.md:103]: I was not aware of this link. Nice.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Bas-Man) [reply CONTRIBUTING.md:101]: Could comment it out for now. Add it back of a different section of help is added?

**Review by BCNelson — APPROVED** [review]

**Pull request merged**